network/templates/base/stations.html

```diff
@@ -33,9 +35,9 @@
               </a>
             </td>
             <td>{{ station.name }}</td>
-            {% if station.location %}
+            {% if station.qthlocator %}
               <td title="{{ station.lat|floatformat:-3 }}, {{ station.lng|floatformat:-3 }}">
-                {{ station.location }}
+                {{ station.qthlocator }}
```

Comment:
I think it's good to have both 'location` and `qthlocator` in the stations view. So, you don't have to replace it. Add an extra column here and above at `<thead>` section.